src/Integrations/Joomla/Joomla.php

```diff
@@ -67,20 +59,20 @@ public function __construct(Container $container, Webservice $webservice)
 		// Constant that is checked in included files to prevent direct access.
 		define('_JEXEC', 1);
 
-		$client = $webservice->options->get('webserviceClient', 'site');
+		$client = $webservice->getOptions()->get('webserviceClient', 'site');
 
 		// Get the CMS base data and load the application
 		if ($client == 'administrator')
 		{
 			define('JPATH_BASE',      JPATH_CMS . DIRECTORY_SEPARATOR . 'administrator');
-			require_once JPATH_BASE . '/includes/defines.php';
+			require_once __DIR__ . '/defines.php';
```

Comment:
It was a quick fix to resolve path conflicts.  Needs revisiting as it doesn't work with the Joomla CMS defines at the moment.